test/functional/feature_rbf.py

```diff
@@ -622,5 +625,31 @@ def test_replacement_relay_fee(self):
         tx.vout[0].nValue -= 1
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
+    def test_fullrbf(self):
+        confirmed_utxo = self.wallet.get_utxo()
```

Comment:
I don't think this is guaranteed to be confirmed, based on the previous tests, unless a block is mined with all outstanding txs before this call? See also the test failure.

Reply:
See comment answering to Jon review. tldr #22698